src/plays/zoomlogin/components/Footer.js

```diff
@@ -0,0 +1,28 @@
+import copyright from '../assets/copyright.svg';
+import linkedin from '../assets/linkedin.svg';
+import github from '../assets/github.svg';
```

Comment:
you could use the icon's cdn link instead of SVG images

Reply:
I changed them to SVG images.